internal/services/cosmos/cosmosdb_postgresql_cluster_resource_test.go

```diff
@@ -263,10 +263,11 @@ resource "azurerm_cosmosdb_postgresql_cluster" "test2" {
   source_location      = azurerm_cosmosdb_postgresql_cluster.test.location
   source_resource_id   = azurerm_cosmosdb_postgresql_cluster.test.id
   point_in_time_in_utc = azurerm_cosmosdb_postgresql_cluster.test.earliest_restore_time
+  node_count           = 0
```

Comment:
why have we removed node count here?

Reply:
I didn't removed "node_count". It's just the format adjusting.